test/common_test/joint_features.cc

```diff
@@ -1296,9 +1296,9 @@ TYPED_TEST(JointFeaturesAttachDetachTest, JointAttachMultiple)
       gz::math::Vector3d body3LinearVelocity =
```

Comment:
nit: why do we need to convert to eigen types first? Doesn't seem like we are using a custom matcher in any case. Would it be enough to check with `frameDataModel1Body.linearVelocity.z()` instead of `body1LinearVelocity.Z()`?

Reply:
we shouldn't need to. I removed these conversions. [8ea3e67](https://github.com/gazebosim/gz-physics/pull/600/commits/8ea3e67e90177a28a5e6ed31da02dbec7bb93415)